empire2018.scss

```diff
@@ -251,6 +253,14 @@ ul {
   line-height: 1.5em;
 }
 
+.sponsor-logo {
+  background: $white;
```

Comment:
I don't love the white against the beige...was the grid behind the logos too busy? (Not a blocker.)

Reply:
So had white and some didn't and some had white but zero padding and it was all v weird